### PR TITLE
Konvert Picasso

### PR DESCRIPTION
**Pull request by jrodbx**
100% Kotlin!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ buildscript {
     errorProneJavac: 'com.google.errorprone:javac:9+181-r4173-1',
     junit: 'junit:junit:4.13.2',
     truth: 'com.google.truth:truth:1.1.3',
-    robolectric: 'org.robolectric:robolectric:4.4',
+    robolectric: 'org.robolectric:robolectric:4.6.1',
```
Comment (jrodbx) [inline build.gradle:32]: Bitmap transformations weren't copying BitmapConfig over.  Bumping Robolectric fixed that.

File: picasso/src/test/java/com/squareup/picasso3/TestUtils.kt [hunk 2]
```diff
@@ -408,4 +409,21 @@ internal object TestUtils {
...
+  }
+
+  class KArgumentCaptor<T>(
+    private val captor: ArgumentCaptor<T>,
+  ) {
+    val value: T
+      get() = captor.value
+
+    fun capture(): T = captor.capture()
+  }
```
Comment (jrodbx) [inline picasso/src/test/java/com/squareup/picasso3/TestUtils.kt:428]: I didn't want to add yet another Mockito dependency, so this unblocks.

**Review by gamepro65 — APPROVED** [review]

File: picasso/src/main/java/com/squareup/picasso3/Picasso.java [hunk 3]
```diff
@@ -1,908 +0,0 @@
...
-
-  /** {@code true} if debug indicators should are displayed on images. */
-  @SuppressWarnings("UnusedDeclaration") public boolean getIndicatorsEnabled() {
-    return indicatorsEnabled;
-  }
-
-  /**
-   * Toggle whether debug logging is enabled.
-   * <p>
-   * <b>WARNING:</b> Enabling this will result in excessive object allocation. This should be only
```
Comment (Egorand) [inline picasso/src/main/java/com/squareup/picasso3/Picasso.java:469]: Don't see this documentation being migrated, might be useful.

**Review by Egorand — APPROVED** [review]

File: picasso/src/main/java/com/squareup/picasso3/Picasso.java (hunk 3 above)
Comment (jrodbx) [reply picasso/src/main/java/com/squareup/picasso3/Picasso.java:469]: good catch!

**Review by chris-horner — APPROVED** [review]

**Pull request merged**